Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.Members.cs

```diff
@@ -35,14 +36,26 @@ public partial class ImageEx
         /// <summary>
         /// Event raised if the image failed loading.
         /// </summary>
+        [Obsolete("This event is obsolete; use ImageFailedEx event instead")]
         public event ExceptionRoutedEventHandler ImageFailed;
 
         /// <summary>
         /// Event raised when the image is successfully loaded and opened.
         /// </summary>
+        [Obsolete("This event is obsolete; use ImageOpenedEx event instead")]
         public event RoutedEventHandler ImageOpened;
 
         /// <summary>
+        /// Event raised if the image failed loading.
+        /// </summary>
+        public event ImageFailedExEventHandler ImageFailedEx;
```

Comment:
Warning, this is my picky moment:)
Should we name it ImageExFailed instead (same for Opened) ?